src/themes/_color_themes.scss

```diff
@@ -80,3 +80,7 @@ $color_themes: (
 @function themed($key) {
   @return map-get($theme-map, $key);
 }
+
+@function themed2($keys...) {
+  @return map-deep-get($theme-map, $keys...);
+}
```

Comment:
Ah, this is a leftover from what I usually want to work on... implementing the font color shades...
It's ok to leave it here, it's currently not in use. I'll work on that later in another feature branch.